### PR TITLE
[docs] Update python style conventions #122

### PR DESCRIPTION
**Pull request by prajwal296**
fixes #112

### CODE REVIEW

**Review by Purhan — CHANGES_REQUESTED** [review]
@prajwal296 your commit message is wrong (referencing the wrong issue). Moreover, someone else was already working on this issue.

File: developer/contributing.rst [hunk 1]
```diff
@@ -158,18 +158,16 @@ OpenWISP follows `PEP 8 -- Style Guide for Python Code
...
-  automatically check the quality of the python code being committed,
-  each python repo has either a ``flake8`` configuration defined in
-  ``setup.cfg`` or a ``runflake8`` script that you can launch with
-  ``flake8`` or ``./runflake8`` respectively.
 - `isort <http://isort.readthedocs.io/en/latest/>`_ is used in order
-  to sort import in a specifc predictable order; each python repo has
-  a ``runisort`` script that you can launch with ``./runisort``
-- Lastly, `black <https://black.readthedocs.io/en/stable/>`_ is used to
+  to sort import in a specifc predictable order.
+- `black <https://black.readthedocs.io/en/stable/>`_ is used to
```
Comment (nemesisdesign) [inline developer/contributing.rst:163]: falke8, isort and black are all used, we need to mention this without mentioning the scripts which are not being used anymore.

File: developer/contributing.rst [hunk 2]
```diff
@@ -158,18 +158,16 @@ OpenWISP follows `PEP 8 -- Style Guide for Python Code
...
   automatically format the code according to our conventions.
 
-For your convenience, we provide the following scripts available in the
+For your convenience, we provide a script in every repository.
+- You can trigger this script with `./run-qa-checks` to initialize 
+  checks for these code conventions.
```
Comment (nemesisdesign) [inline developer/contributing.rst:168]: Add this after the mention to openwisp-qa-format, please explain that these scripts contain the logic to perform all the QA checks specific to each module

**Review by nemesisdesign — CHANGES_REQUESTED** [review]

**Comment by Purhan** [comment]
Seems the issue is closed. Reopen if not.

**Pull request closed without merging**